src/index.md

```diff
@@ -10,9 +10,9 @@ pagination:
 <ul class="posts-ul">
   <% paginator.resources.each do |post| %>
     <li class="posts-list-item">
-      <a class="width-max-content" href="<%= post.relative_url %>">
```

Comment:
Deleting this `width-max-content` class is the key change to prevent horizontal scrolling.